src/pluto/pluto.py

```diff
@@ -17,72 +16,31 @@ def pluto():
     """
     )
 
-    engine = create_engine(ENGINE)
-
-    @st.cache(suppress_st_warning=True, allow_output_mutation=True)
-    def get_data():
-        def convert(dt):
-            try:
-                d = datetime.strptime(dt, "%Y/%m/%d")
-                return d.strftime("%m/%d/%y")
-            except BaseException:
-                return dt
-
-        engine = create_engine(ENGINE)
-        df_mismatch = pd.read_sql("SELECT * FROM dcp_pluto.qaqc_mismatch", con=engine)
-        df_null = pd.read_sql("SELECT * FROM dcp_pluto.qaqc_null", con=engine)
-        df_aggregate = pd.read_sql("SELECT * FROM dcp_pluto.qaqc_aggregate", con=engine)
-        source_data_versions = pd.read_csv(
-            "https://edm-publishing.nyc3.digitaloceanspaces.com/db-pluto/latest/output/source_data_versions.csv"
-        )
-        sdv = source_data_versions.to_dict("records")
-        version = {}
-        for i in sdv:
-            version[i["schema_name"]] = i["v"]
-        version_text = f"""
-            Department of City Planning – E-Designations: ***{convert(version['dcp_edesignation'])}***  
-            Department of City Planning – Georeferenced NYC Zoning Maps: ***{convert(version['dcp_zoningmapindex'])}***  
-            Department of City Planning – NYC City Owned and Leased Properties: ***{convert(version['dcp_colp'])}***  
-            Department of City Planning – NYC GIS Zoning Features: ***{convert(version['dcp_zoningdistricts'])}***  
-            Department of City Planning – Political and Administrative Districts: ***{convert(version['dcp_cdboundaries_wi'])}***  
-            Department of City Planning – Geosupport version: ***{convert(version['dcp_cdboundaries_wi'])}***  
-            Department of Finance – Digital Tax Map (DTM): ***{convert(version['dof_dtm'])}***  
-            Department of Finance – Mass Appraisal System (CAMA): ***{convert(version['pluto_input_cama_dof'])}***  
-            Department of Finance – Property Tax System (PTS): ***{convert(version['pluto_pts'])}***  
-            Landmarks Preservation Commission – Historic Districts: ***{convert(version['lpc_historic_districts'])}***  
-            Landmarks Preservation Commission – Individual Landmarks: ***{convert(version['lpc_landmarks'])}***  
-        """
-        log = requests.get(
-            "https://raw.githubusercontent.com/NYCPlanning/db-pluto/master/maintenance/log.md"
-        ).text
-        return (
-            df_mismatch,
-            df_null,
-            df_aggregate,
-            source_data_versions,
-            version_text,
-            log,
-        )
+    branches = get_branches()
+    branch = st.sidebar.selectbox(
+        "select a branch",
+        branches,
+        index=branches.index("317-QAQC-to-DO"),
```

Comment:
yes for sure this will be changed once the qaqc tables are sent to main, which should happen in [this build](https://github.com/NYCPlanning/db-pluto/actions/runs/2664592879)